subsys/net/ip/net_stats.h

```diff
@@ -318,27 +318,21 @@ static inline void net_stats_update_ipv6_mld_drop(struct net_if *iface)
 #define net_stats_update_ipv6_mld_drop(iface)
 #endif /* CONFIG_NET_STATISTICS_MLD */
 
-#if defined(CONFIG_NET_CONTEXT_TIMESTAMP) && defined(CONFIG_NET_STATISTICS)
+#if (defined(CONFIG_NET_CONTEXT_TIMESTAMP) || \
+	defined(CONFIG_NET_PKT_TXTIME_STATS)) && defined(CONFIG_NET_STATISTICS)
 static inline void net_stats_update_tx_time(struct net_if *iface,
 					    u32_t start_time,
 					    u32_t end_time)
 {
-	u32_t diff = abs(end_time - start_time);
```

Comment:
Oh, thanks for fixing for the older cases. You might see why I was sour with my comment - because that issue already surfaced (I definitely remember it), and that time, my suggestions were of no avail.